test/error-cat.js

```diff
@@ -118,17 +118,19 @@ describe('ErrorCat', function() {
 
     beforeEach(function (done) {
       sinon.stub(error, 'log');
+      sinon.stub(error, 'report');
       sinon.stub(Boom, 'create');
       done();
     });
 
     afterEach(function (done) {
       error.log.restore();
+      error.report.restore();
       Boom.create.restore();
       done();
     });
 
-    it('should create a new boom error', function(done) {
+    it('should create a new boom error, not send to rollbar', function(done) {
```

Comment:
I understand what it is doing, I was referring to the wording (isn't it a little weird and ambiguous?)